.github/scripts/get_product_version.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
For versioning to be compatible with CRT version bump automation (automatically bumping patches on releases + cutting new minor version branches automatically), we would need versioning to follow our standard release branch format. This would look like developing on a branch `release/1.x.x` and versioning to look like `1.x.x-alpha` instead of including git tag information. Is this something y'all want/would be willing to do?

Reply:
I think we'd prefer to put off that decision if we can, in the interests of not breaking our current release pipeline while we're making the transition to CRT! Is there any downside to keeping our current branching/versioning scheme for now?

Reply:
The only downside is not having version bump automation (ie automatically bumping versions after releases)! I think eventually we'd want to standardize on version stuff, but it's non-urgent.

Reply:
Cool! We're definitely open to change there, but there's some assumptions about the release branch formatting built into our current release pipeline so I'd just as soon wait until we feel confident in our CRT releases before changing.